apps/services/services.py

```diff
@@ -40,3 +40,20 @@ async def create_user_document(**kwargs):
                                         content=content,
                                         user_request_id=kwargs.get("user_request_id"))
     return await database.execute(query)
+
+
+def calculate_dates_different(start_date: date, end_date: date) -> int:
+    if end_date <= start_date:
+        raise ValueError(
+            message="Start date of hostel accommodation can't be more or equal than end date of hostel accommodation"
+        )
+    return -12 * (start_date.year - end_date.year) - start_date.month + end_date.month
+
+
+def get_month_price_by_bed_place(hostel_month_price: float, bed_place_name: str) -> float:
+    return hostel_month_price * float(bed_place_name)
+
+
+def calculate_hostel_accommodation_cost(month_price: float, dates_different: int) -> float:
```

Comment:
1) rename method to calculate_total_hostel_accommodation_cost
2) months instead of dates_different
3) returned type is decimal, not float
4) please dates_different rename it to month_count